src/north/CsvToHttp/CsvToHttp.class.js

```diff
@@ -174,12 +267,47 @@ class CsvToHttp extends ApiHandler {
           object.error.push(`Header "${mapping.httpField}": ${response.error}`)
         }
         object.value[field] = response.value
+      } else if (mapping.csvField.match(REGEX_CONTAIN_VARIABLE_STRING)) {
+        // split the input into a array of string
+        // "test ${value}" => [ "test ", ${value}]
+        const csvFieldSplit = mapping.csvField.split(REGEX_SPLIT_TEMPLATE_STRING).filter(Boolean)
```

Comment:
filter is not useful here

Reply:
I think it's useful because in the REGEX we have a "or" logic  which means that without it we can have empty string sent

Reply:
My bad, ok for this one !